packages/block-library/src/gallery/block.json

```
@@ -102,12 +102,17 @@
 		"allowResize": {
 			"type": "boolean",
 			"default": false
+		},
+		"aspectRatio": {
+			"type": "string",
+			"default": "auto"
 		}
 	},
 	"providesContext": {
 		"allowResize": "allowResize",
 		"imageCrop": "imageCrop",
-		"fixedHeight": "fixedHeight"
+		"fixedHeight": "fixedHeight",
+		"aspectRatio": "aspectRatio"
```

Why do we need to provide the context?

---

i can't see a reason, so let's remove it. I've tested and it works with it.
```suggestion
		"fixedHeight": "fixedHeight"
```